src/Book/src/Entity/Book.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Light\Book\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Light\App\Entity\AbstractEntity;
+use Light\Book\Repository\BookRepository;
+
+#[ORM\Entity(repositoryClass: BookRepository::class)]
+#[ORM\Table(name: 'books')]
+#[ORM\HasLifecycleCallbacks]
+class Book extends AbstractEntity
+{
+    #[ORM\Column(name: 'title', type: 'string', length: 500, nullable: true)]
+    private ?string $title = null;
+
+    #[ORM\Column(name: 'author', type: 'string', length: 500, nullable: true)]
+    private ?string $author = null;
+
+    public function __construct()
```

Comment:
If a child constructor does nothing more than calling the parent constructor, then the child constructor can be omitted.